docs/operate/get-started/other-hardware/_index.md

```diff
@@ -1007,6 +1007,12 @@ Do not change the <code>module_id</code>.</p>
 <td>An object containing the command to run to build your module, as well as optional fields for the path to your dependency setup script, the target architectures to build for, and the path to your built module. Use this with the <a href="/dev/tools/cli/#using-the-build-subcommand">Viam CLI's build subcommand</a>. </td>
 </tr>
 <tr>
+<td><code>markdown_link</code></td>
```

Comment:
I think this will almost certainly be overlooked here and it's crucial we get people to fill this one in. If we don't our AI will now understand the module and its configuration.

Reply:
Okay--talking to Grant about getting this to auto-populate like the model markdown links. In the meantime, adding half a sentence up top. I know we don't want to add too many alert boxes...is there something more you'd like me to do here other than follow up about auto-population?

Reply:
Grant is making it auto-populate in next week's deployment. Thanks Grant!

Reply:
For future reference, is there a way you would have liked me to surface it differently if it wasn't going to autopopulate?